addons/web/static/src/views/calendar/calendar_common/calendar_common_popover.js

```diff
@@ -25,6 +26,28 @@ export class CalendarCommonPopover extends Component {
         return this.props.model.canDelete;
     }
 
+    getFormattedValue(fieldName) {
+        const fieldInfo = this.props.model.popoverFields[fieldName];
+        const field = this.props.model.fields[fieldName];
+        let format;
+        let value = this.props.record.rawRecord[fieldName];
+        if (["many2many", "one2many"].includes(field.type)) {
+            // needed since the formatX2many need a object with `currentIds` as attribute.
+            value = {
+                currentIds: value,
+            };
+        } else if (["date", "datetime"].includes(field.type)) {
+            return value;
+        }
```

Comment:
Coucou @mcm-odoo 👋🏼 

Thanks again for your review!

I have to do that since we have raw values compared to kanban/list view. 🙈 

I hope it is okay for you. 🙂

Reply:
Hello, you can get the right values from `slot.record.data` in the template. I would give the record to this function.